microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -332,75 +380,87 @@ For more information about the visual **Font Editor**, see [Edit CSS font styles
-*  This checkbox is present in Microsoft Edge Canary v104.
+*  This checkbox is present in Microsoft Edge Stable v102.
+*  This checkbox is present in Microsoft Edge Canary v105.
 
 
 <!-- ====================================================================== -->
 ## Enable experimental cookie features
 
 Enables experimental cookie features.
-<!-- Needs content. -->
+<!-- todo: need text -->
 
-*  This checkbox is present in Microsoft Edge Stable v101.
-*  This checkbox is present in Microsoft Edge Canary v104.
+*  This checkbox is present in Microsoft Edge Stable v102.
+*  This checkbox is present in Microsoft Edge Canary v105.
 
 
 <!-- ====================================================================== -->
 ## Enable Reporting API panel in the Application panel
 
 Use the Reporting API to catch certain errors such as security violations or deprecated API calls. These errors happen when users visit your site and are sent to a server endpoint. Enable this experiment to add the **Reporting API** section in the **Application** panel, which lists all of the reports sent to the endpoint.
 
-*  This checkbox is present in Microsoft Edge Stable v101.
-*  This checkbox is present in Microsoft Edge Canary v104.
+*  This checkbox is present in Microsoft Edge Stable v102.
+*  This checkbox is present in Microsoft Edge Canary v105.
 
 
 <!-- ====================================================================== -->
 ## Display more precise changes in the Changes tab
 
 See [More precise changes in the Changes tab](https://developer.chrome.com/blog/new-in-devtools-98/#changes).
 
-*  This checkbox is present in Microsoft Edge Stable v101.
-*  This checkbox is present in Microsoft Edge Canary v104.
+*  This checkbox is present in Microsoft Edge Stable v102.
+*  This checkbox is present in Microsoft Edge Canary v105.
 
 
 <!-- ====================================================================== -->
 ## Sync CSS changes in the Styles pane
 
 Whether to sync CSS changes in the **Styles** tab in the **Elements** tool.
-<!-- Needs content. -->
+<!-- todo: need text -->
 
-*  This checkbox is present in Microsoft Edge Stable v101.
-*  This checkbox is present in Microsoft Edge Canary v104.
+*  This checkbox is present in Microsoft Edge Stable v102.
+*  This checkbox is present in Microsoft Edge Canary v105.
 
 
 <!-- ====================================================================== -->
 ## Local overrides for response headers
 
 Whether to use local overrides for response headers.
-<!-- Needs content. -->
+<!-- todo: need text -->
+
+*  This checkbox is present in Microsoft Edge Stable v102.
+*  This checkbox is present in Microsoft Edge Canary v105.
+
 
-*  This checkbox is present in Microsoft Edge Stable v101.
-*  This checkbox is present in Microsoft Edge Canary v104.
+<!-- ====================================================================== -->
+## Enable CSS Authoring hints for inactive rules, deprecated properties, etc.
```

Comment:
@captainbrosset  @zoherghadyali - need text for v105 Experiments checkbox "Enable CSS Authoring hints for inactive rules, deprecated properties, etc."